### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by Mahendra114027**
Updating md to new terms

### CODE REVIEW

File: VTU/Sem6/USP_CD_Lab/08AvoidZombie/08_avoid_zombie.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
-##Aim:
+## Aim:
 
-###Write a C/C++ program to avoid zombie process by forking twice.
+###W rite a C/C++ program to avoid zombie process by forking twice.
```
Comment (cyberhck) [inline VTU/Sem6/USP_CD_Lab/08AvoidZombie/08_avoid_zombie.md]: Please fix this one, you mistakenly have a space in wrong place

File: VTU/Sem6/USP_CD_Lab/10AlarmSignalHandler/10_alarm_signal_handler.md [hunk 2]
```diff
@@ -1,13 +1,13 @@
-##Aim :
-###Write a C/C++ program to set up a real-time clock interval timer using the alarm API.
+## im :
```
Comment (cyberhck) [inline VTU/Sem6/USP_CD_Lab/10AlarmSignalHandler/10_alarm_signal_handler.md]: A is gone, I don't believe that to be intensional

**Comment by cyberhck** [comment]
Thanks @Mahendra114027

**Comment by Mahendra114027** [comment]
Thanks @cyberhck  for honest comments

**Pull request merged**